src/main/java/nl/knmi/geoweb/backend/product/airmet/Airmet.java

```diff
@@ -512,6 +513,7 @@ public Airmet(Airmet otherAirmet) {
         this.issuedate = otherAirmet.getIssuedate();
         this.firFeature = otherAirmet.firFeature;
         this.type=otherAirmet.type;
+        this.change= otherAirmet.change;
```

Comment:
I think this should be corrected, for two reasons:
* change is a private member, so it should be otherAirmet.getChange()
* at line 502 the change is already set